content/swiftui/concepts/gestures/gestures.md

```diff
@@ -0,0 +1,137 @@
+---
+Title: 'Gestures' 
+Description: 'Defines interactions from taps, clicks, and swipes to fine-grained gestures.' 
+Subjects: 
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'SwiftUI ViewModifiers'
+  - 'SwiftUI'
+  - 'iOS'
+CatalogContent: 
+  - 'learn-iOS-course'
+  - 'path/iOS-path'
+---
+
+`Views` can respond to taps, drags, pinches and other gestures just by adding **Gesture** modifiers to the view.
+
+## Tap Gestures
+
+This is the most basic type of Gesture.
+The `.onTapGesture()` modifier can be called on the view desired to track the gesture on.
+
+```Swift
+struct Tap: View {
+    var body: some View {
+        Text("Hello, World!")
+            .onTapGesture {
+                print("View Tapped") 
+            }
+    }
+}
+```
+
+The output looks like:
+
+![This is a gif that shows the effect of using TapGesture in Xcode simulation](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-tapgesture-example.gif)
+
+Every time the text view "Hello, World" is tapped, it prints "View Tapped" in the console.
+The action can also be implemented after certain amount of tap gestures. For example:
+
+```Swift
+struct Tap: View {
+    var body: some View {
+        Text("Hello, World!")
+            .onTapGesture(count: 3) {
+                print("View Tapped") 
+            }
+        
+    }
+}
+```
+In this example it prints "View Tapped" after taping the text view three times.
+
+> **Note:** When a button is needed, use a Button instance rather than a tap gesture. Any view can be used as the button’s label, and the button type automatically provides many of the standard behaviors that users expect from a button, like accessibility labels and hints. 
+
+## Long Press Gestures
+
+`onLongPressGesture()` it's used for handling long presses gesture. `onLongPressGesture()` have an extra parameter called `minimumDuration:` (takes Double as input), it's used to specify after how much time the action will be executed. For example:
+
+```Swift
+struct Tap: View {
+    var body: some View {
+        Text("Hello, World!")
+            .onLongPressGesture(minimumDuration: 2) {
+                print("View Tapped")
+            }
+        
+    }
+}
+```
+The output looks like:
+
+![This is a gif that shows the effect of using LongPressGesture in Xcode simulation](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-longgesture-example.gif)
+

```

Comment:
```suggestion
A string will be printed in the console after pressing the text view for 2 seconds. On Xcode, this will display:

![This is a gif that shows the effect of using LongPressGesture in Xcode simulation](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-longgesture-example.gif)
```